purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -26,15 +20,10 @@ import com.revenuecat.purchases.interfaces.UpdatedCustomerInfoListener
 import com.revenuecat.purchases.models.BillingFeature
 import com.revenuecat.purchases.models.GoogleProrationMode
 import com.revenuecat.purchases.models.StoreProduct
-import com.revenuecat.purchases.models.StoreTransaction
-import com.revenuecat.purchases.models.SubscriptionOption
 import com.revenuecat.purchases.strings.BillingStrings
 import com.revenuecat.purchases.strings.ConfigureStrings
 import java.net.URL
 
-typealias SuccessfulPurchaseCallback = (StoreTransaction, CustomerInfo) -> Unit
-typealias ErrorPurchaseCallback = (StoreTransaction, PurchasesError) -> Unit
```

Comment:
I kept these in the `main` source set... I don't think these should be public, but we use them internally, so they would need to be public in `defaults` to not cause a breaking change but internal in `customEntitlementComputation`.